client/src/graphql/join-group.mutation.js

```diff
@@ -6,25 +6,34 @@ export default gql`
       id
       name
       tags {
-         name
-         id
-       }
-       users {
-         username
-         id
-       }
-       events {
-         id
-         name
-         details
-       }
-       schedules {
-         id
-         name
-         details
-         startTime
-         endTime
-       }
+        name
+        id
+      }
+      users {
+        username
```

Comment:
can you confirm we dont need displayName here?

Reply:
We should include it if we need it - I'll update it